src/views/CreateMeetup.vue

```diff
@@ -1,11 +1,149 @@
 <template>
-  <h1>Create meetup</h1>
+  <v-form
+    class="mx-auto"
+    style="max-width: 600px"
+    @submit.prevent="createMeetup"
+  >
+    <h2 class="pt-5 grey--text text--lighten-1 font-weight-regular">
+      Create a new Meetup
+    </h2>
+
+    <v-text-field
+      label="Title*"
+      required
+      color="dark lighten-1"
+      v-model="title"
+    ></v-text-field>
+
+    <v-text-field
+      label="Location*"
+      required
+      color="dark lighten-1"
+      v-model="location"
+    ></v-text-field>
+
+    <v-btn
+      color="dark lighten-1"
+      class="text-none mb-5"
+      :loading="selectedImg.isSelecting"
+      @click="onUploadImgBtnClick()"
+    >
+      UPLOAD IMAGE
+    </v-btn>
+    <input
+      ref="uploader"
+      class="d-none"
+      type="file"
+      accept="image/*"
+      @change="onFileChanged"
+    >
+    <v-img
+      v-if="selectedImg.img"
+      :src="selectedImg.url"
+      max-width="300"
+      max-height="300"
+      class="rounded"
+    ></v-img>
+
+    <v-textarea
+      color="dark lighten-1"
+      class="mt-5"
+      label="Description*"
+      no-resize
+      v-model="description"
+    ></v-textarea>
+
+    <h2 class="pt-5 grey--text text--lighten-1 font-weight-regular">
+      Choose a Data & Time
+    </h2>
+
+    <v-date-picker
+      v-model="meetupData"
+      :min="nowDate"
+      class="ma-1"
+      width="285"
+      elevation="3"
+      color="grey lighten-1"
+      style="height: 400px"
+    ></v-date-picker>
+
+    <v-time-picker
+      elevation="3"
+      v-model="meetupTime"
+      format="24hr"
+      class="ma-1"
+      width="285"
+      color="grey lighten-1"
+      style="height: 400px"
+    ></v-time-picker>
+
+    <br>
+    <v-btn
+      color="dark"
+      class="mt-10 mb-10"
+      type="submit"
+    >CREATE MEETUP</v-btn>
+  </v-form>
 </template>
 
 <script>
 // @ is an alias to /src
 
 export default {
   name: 'CreateMeetup',
+  data: () => ({
+    title: '',
+    location: '',
+    selectedImg: {
+      img: null,
+      url: '',
+      isSelecting: false,
+    },
+    description: '',
+    meetupData: '',
+    meetupTime: '09:00',
+    nowDate: new Date().toISOString().slice(0, 10),
```

Comment:
You are using this variable to set min property of date-picker, but it should be tomorrow date. So I believe that you don't need this extra variable, use meetupData to set up min property of date-picker

Reply:
This solution didn't work for me https://www.screencast.com/t/BjrJFecvSQ I used another instead